src/coreclr/nativeaot/Runtime/clrgc.disabled.cpp

```diff
@@ -0,0 +1,17 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+#ifdef _WIN32
+// this must exactly match the typedef used by windows.h
```

Comment:
Can we include whatever defines HRESULT instead of duplicating the definition here?